bower.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "html-template",
```

Comment:
Just call it `template`. This won't be published to the bower catalog, so the name doesn't matter for uniqueness